### PR TITLE
Alt Text Inclusions

### PR DESCRIPTION
**Pull request by dereklh7**
Added missing alt text.

### CODE REVIEW

File: iis/configuration/system.applicationHost/sites/site/applicationDefaults.md [hunk 1]
```diff
@@ -57,11 +57,11 @@ The `<applicationDefaults>` element of the `<site>` element is included in the d
         - Double-click **Administrative Tools**, and then double-click **Internet Information Services (IIS) Manager**.
 2. In the **Connections** pane, expand the server name, expand the **Sites** node, then click the name of the site.
 3. In the site's **Home** pane, click **View Applications** in the **Actions** pane.  
-    [![](applicationDefaults/_static/image2.png)](applicationDefaults/_static/image1.png)
+    [![Screenshot of the Default Web Site Home screen, showing the dot NET Compilation, dot NET Globalization, and dot NET Profile options.](applicationDefaults/_static/image2.png)](applicationDefaults/_static/image1.png)
```
Comment (TimShererWithAquent) [inline iis/configuration/system.applicationHost/sites/site/applicationDefaults.md:60]: Is this called a "screen" anywhere?

File: iis/web-hosting/verifying-the-deployment/verifying-remote-administration.md [hunk 2]
```diff
@@ -30,17 +30,17 @@ An important characteristic of [Shared Hosting](../planning-the-web-hosting-arch
 
 1. Within a Windows Server 2008 R2, Windows 7, or Windows Vista computer run InetMgr.exe.
 2. From the Connections pane select Connect to Site:  
-    [![](verifying-remote-administration/_static/image2.jpg)](verifying-remote-administration/_static/image1.jpg)
+    [![Screenshot of the Connections pane with the Connect to a Site option in the right-click drop-down menu being highlighted.](verifying-remote-administration/_static/image2.jpg)](verifying-remote-administration/_static/image1.jpg)
```
Comment (TimShererWithAquent) [inline iis/web-hosting/verifying-the-deployment/verifying-remote-administration.md:33]: "dropdown" is one word in the writing guide https://styleguides.azurewebsites.net/Styleguide/Read?id=2700  - not sure if that's still current.

**Comment by dereklh7** [comment]
@Rick-Anderson @mcleblanc Final review for adding alt text.

File: iis/web-hosting/verifying-the-deployment/verifying-remote-administration.md (hunk 2 above)
Comment (Rick-Anderson) [reply iis/web-hosting/verifying-the-deployment/verifying-remote-administration.md]: > "dropdown" is one word in the writing guide https://styleguides.azurewebsites.net/Styleguide/Read?id=2700 - not sure if that's still current.

Yes, still one word.

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**